.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
       - '*'
 
 jobs:
-  tests:
+  library:
```

Comment:
this updates CI to run library tests on 11.3/11.7/12.4 and run example app tests only on 11.7/12.4 so that we can use key path functions.